articles/language/statements.md

```diff
@@ -235,7 +233,7 @@ Correspondingly, update-and-reassign statements exist for named items in user-de
 ```qsharp
 newtype Complex = (Re : Double, Im : Double);
 
-function AddAll (reals : Double[], ims : Double[]) : Complex[] {
+function AddAll(reals : Double[], ims : Double[]) : Complex[] {
```

Comment:
Shouldn't it be a name since it is a function? Maybe "AdditionAll"?

Reply:
I tried ComplexAddition, maybe that works?